File: src/main/java/com/sara/Hello.java
package com.sara;

public class Hello {
    public static void main(String[] args) {
//        System.out.println("Hello world");
        Person p = new Person(66.5f, 1.7f);
        p.hello();

        p.weight = 66.5f;
        p.height = 1.7f;
        System.out.println(p.bmi());
    }
}

File: src/main/kotlin/com/sara/Hello.kt
package com.sara

fun main(args: Array<String>) {
//    println("Hello Kotlin")
//    Human().hello()
    var h = Human(weight = 66.5f, height = 1.7f)
    println(h.bmi())
//    var name : String
//    name = "Hank"
}

class Human(var weight : Float, var height : Float) {

    fun bmi() : Float {
        val bmi = weight / (height*height)
        return bmi
    }

    fun hello() {
        println("Hello Kotlin")
    }
}